packages/salesforcedx-apex-debugger/src/adapter/apexDebug.ts

```diff
@@ -664,7 +1379,7 @@ export class ApexDebug extends DebugSession {
       logMessage += ` |${os.EOL}${message.sobject.Stacktrace}`;
     }
 
-    this.logToDebugConsole(
+    this.printToDebugConsole(
```

Comment:
Why the name change? I thought logToDebugConsole was OK.

Reply:
I thought the name would make it more clear that this is not the preferred way of logging/tracing in the code but more a utility method for printing something to the DebugConsole.